gapis/api/templates/README.md

```diff
@@ -48,19 +48,6 @@ being the faster option:
     {{Template "Foocakes" "Hello world"}}
     ```
 
-## File macros
```

Comment:
To clarify, is this documentation removed because the File macro is never used?

Reply:
There is no File macro. Maybe there once was, but there isn't now.